pyanaconda/modules/subscription/subscription.py

```diff
@@ -364,3 +368,25 @@ def set_connect_to_insights(self, connect):
         self._connect_to_insights = connect
         self.connect_to_insights_changed.emit()
         log.debug("Connect target system to Insights set to: %s", self._connect_to_insights)
+
+    # subscription status
+
+    @property
+    def subscription_attached(self):
+        """Return True if a subscription has been attached to the system.
+
+        :return: True if a subscription has been attached to the system, False otherwise
+        :rtype: bool
+        """
+        return self._subscription_attached
+
+    def set_subscription_attached(self, system_subscription_attached):
+        """Set a subscription has been attached to the system.
+        :param bool system_registered: True if subscription has been attached, False otherwise
```

Comment:
Missing an empty space between doc title and description.

Reply:
Fixed. :)